### ISSUE TITLE
Themes not working on publish

### ISSUE THREAD

**Issue by iagocaldeira** [jsonresume/resume-cli#158]
Only 'flat' theme is working under settings on cli, neither modern and elegant are.
This is rellated to *Theme error [bug]* #148
I can't even publish with other themes like slick.

```
Theme returned an error.
TypeError: Cannot read property 'profiles' of undefined
    at Object.render (/home/ubuntu/theme-manager/themes/jsonresume-theme-modern/0.0.18/index.js:39:28)
    at runTheme (/home/ubuntu/theme-manager/lib/theme.js:23:24)
    at /home/ubuntu/theme-manager/lib/theme.js:112:13
    at FSReqWrap.cb [as oncomplete] (fs.js:228:19)
Theme Info

{
  "name": "jsonresume-theme-modern",
  "version": "0.0.18",
  "description": "jsonresume-theme-modern =======================",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/jsonresume/jsonresume-theme-modern.git"
  },
  "author": "",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/jsonresume/jsonresume-theme-modern/issues"
  },
  "homepage": "https://github.com/jsonresume/jsonresume-theme-modern",
  "dependencies": {
    "gravatar": "^1.0.6",
    "lodash": "^2.4.1",
    "mustache": "^0.8.2",
    "tarball-extract": "0.0.3"
  }
}
We just launched the official 0.0.0 version recently, please make sure you update before continuing.

Resume.json Validation Test

null

```

**Comment by craigske** [comment]
This is likely your issue: (https://github.com/jsonresume/registry-server/issues/75)

TL;DR:

"basics" should be an array in []

**Comment by chrisdotcode** [comment]
(duplicate of jsonresume/registry-server#75. Thanks, @craigske!)
